config/locales/pt-BR.yml

```diff
@@ -0,0 +1,175 @@
+pt-BR:
+  title: "RubyGems.org"
+  subtitle: "o host de gems da sua comunidade"
+  help_url: "http://help.rubygems.org"
+  please_sign_up: "Accesso Negado. Por favor se registre em: https://rubygems.org"
```

Comment:
Sorry, but I don't see it  https://github.com/rubygems/rubygems.org/blob/ac8c068d568bececabe138eb8d7e4aeef9488f04/config/locales/en.yml#L5